src/faebryk/exporters/netlist/graph.py

```diff
@@ -164,9 +162,7 @@ def name(self) -> str:
         Prefixes and suffixes are joined with a "-" if they exist.
         """
         return "-".join(
-            str(n)
-            for n in [self.prefix, self.base_name or "net", self.suffix]
-            if n is not None
+            str(n) for n in [self.prefix, self.base_name or "net", self.suffix] if n
```

Comment:
This should test "truthiness", rather than is False, because we also don't want an empty string